src/main/java/com/cgm/infolab/IsDevOrTestCondition.java

```diff
@@ -0,0 +1,18 @@
+package com.cgm.infolab;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.env.Environment;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class IsDevOrTestCondition implements Condition {
+    @Override
+    public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+        Environment  env = context.getEnvironment();
+        List<String> profiles = Arrays.asList(env.getActiveProfiles());
+        return profiles.contains("dev") || profiles.contains("test");
```

Comment:
`dev` e `test` ci facciamo delle costanti? sono usati un po' in tanti posti adesso